<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveBestPatchesWidget</class>
 <widget class="QMainWindow" name="InteractiveBestPatchesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interactive Best Patches</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Number of best patches:</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtNumberOfPatches">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;font color=&quot;red&quot;&gt;Target Patch Corner&lt;/font&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtTargetX">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtTargetY">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Patch Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtPatchRadius">
              <property name="toolTip">
               <string>Sets the width/height of a square patch</string>
              </property>
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="chkShowMask">
            <property name="text">
             <string>Show mask</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCompute">
            <property name="text">
             <string>Compute</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKWidget" name="qvtkWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,5,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Target patch:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="gfxTarget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Best source patches:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkFillPatch">
            <property name="text">
             <string>Fill patch</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Patch</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Index</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total Absolute</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Average Absolute</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total Squared</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Average Squared</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Sort by:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radTotalAbsolute">
            <property name="text">
             <string>Total Absolute</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radAverageAbsolute">
            <property name="text">
             <string>Average Absolute</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radTotalSquared">
            <property name="text">
             <string>Total Squared</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radAverageSquared">
            <property name="text">
             <string>Average Squared</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnResort">
            <property name="text">
             <string>Re-sort</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1374</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLeft_Pane">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenMask"/>
    <addaction name="actionOpenMaskInverted"/>
    <addaction name="actionFlipImage"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuLeft_Pane"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSavePointsLeft">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionSavePointsRight">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionOpenImageRight">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionLoadPointsLeft">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionLoadPointsRight">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenPointCloudLeft">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionOpenPointCloudRight">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionFlipImage">
   <property name="text">
    <string>Flip Image</string>
   </property>
  </action>
  <action name="actionFlipRightImage">
   <property name="text">
    <string>Flip Right Image</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionOpenMask">
   <property name="text">
    <string>Open Mask</string>
   </property>
  </action>
  <action name="actionOpenMaskInverted">
   <property name="text">
    <string>Open Mask Inverted</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
